vignettes/function-flow.Rmd

```diff
@@ -121,38 +121,50 @@ input to extract the dates of all diabetes-specific podiatrist services.
 
 <!-- TODO: Add details on how this filtering should be done -->
 
+AAI: By date
+
 #### GLD purchases
 
 The function `include_gld_purchases()` uses `lmdb` to extract the dates
-of all GLD purchases (from 1997 onwards).
-
-<!-- TODO: Add details on how this filtering should be done -->
-
-<!-- TODO: Add this + link to resource "For details about this, see [link]." -->
+of all GLD purchases.
+
+These dates are extracted by filtering values beginning with "A10" in
+the `atc` column of the `lmdb` register. In addition to the identifier
+variable (`pnr`) and date (`eksd`), additional information needed for
+censoring or for classification of diabetes type are also extracted: the
+type of drug (`atc`), the amount purchased (`volume` and `apk`), the
+indication code (`indo`), and its brand name or vnr-number (`name` or
+`vnr`). These events are then passed to a chain of exclusion functions:
+`exclude_wld_purchases()`, `exclude_potential_pcos()`,
+`exclude_pregnancy()` described in the sections below.
+
+After these exclusion functions have been applied, the output serves as
+inputs to two sets of functions:
+
+1.  the `get_diagnosis_date()` function for the final step of the
+    inclusion process.
+2.  the `get_only_insulin_purchases()`,
+    `get_insulin_purchases_within_180_days()`, and
+    `get_insulin_is_two_thirds_of_gld_doses()` helper functions for the
+    classification of diabetes type.
+
+Since the diagnosis code data on pregnancies is insufficient to perform
+censoring prior to 1997, `include_gld_purchases()` only extracts dates
+from 1997 onward by default (if Medical Birth Register data is available
+to use for censoring, the extraction window can be extended).
```

Comment:
Does this mean that we need an argument with a default of 1997-*newest data* that can be changed to include data prior to 1997? 
Or do we not want to include such an argument just yet, but maybe further down the line?

Reply:
Yes. And yes, I think it's fine to include it now, since it's just a default argument.